docker/Dockerfile.server

```diff
@@ -14,4 +14,4 @@ COPY sematic-*.whl .
 RUN pip install sematic-*.whl
 
 EXPOSE 80
-CMD python3 -m sematic.db.migrate up --verbose --env cloud; python3 -m sematic.api.server --env cloud
+CMD python3 -m sematic.db.migrate up --verbose --env cloud && python3 -m sematic.api.server --env cloud
```

Comment:
I realized when I saw this code that it would be risky to have multiple replicas of the server so long as this is the entrypoint; it could lead to multiple replicas all trying to run migrations at once. Created #336 . cc: @snoshy

Reply:
Trying to remember how we did this at Instacart. It was the pre-k8s era, I think we shipped features requiring migrations behind feature flags, deployed them to prod, then ran the migration script by hand (or by CD pipeline), then turned feature flags on. I'm sure there is a more modern and canonical way to do this in the k8s world.